node/cli/src/chain_spec.rs

```diff
@@ -398,7 +399,11 @@ pub(crate) mod tests {
 		service_test::connectivity(
 			integration_test_config_with_two_authorities(),
 			|config| new_full(config),
-			|config| new_light(config),
+			|mut config| {
+				// light nodes are unsupported
```

Comment:
This should also be a TODO.  If this were not a test case, I would prefer to bail out and exit with an error here.

Reply:
disagree - we haven't added light client support as a feature yet. An issue would be fine; I don't see the need for a TODO